src/APICalls/APICalls.js

```diff
@@ -0,0 +1,53 @@
+const theEndpoint = `https://bangwords-api.herokuapp.com/`;
+
+const theCalls = {
+    joinGame(role) {
+        let theBody = {
+            act: "join",
+            method: 'POST',
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify(role)
+        }
+        return fetch(`${theEndpoint}`, theBody)
+            .then(res => res.json())
+            .then(res => console.log(res))
+            .catch(err => console.log(err))
+    },
+    createWord(word, userID) {
+        let theBody = {
+            act: "word",
+            word: `${word}`,
+            id: `${userID}`,
+            method: 'POST',
+            headers: {
+                "Content-Type": "application/json"
+            },
+            body: JSON.stringify(word)
+        }
+        return fetch(`${theEndpoint}`, theBody)
+            .then(res => res.json())
+            .then(res => console.log(res))
+            .catch(err => console.log(err))
+    },
+    makeGuess(guess, userID) {
+        // {act: "guess", guess: "", id:}	{ display: '_ _ _ _', guesses: 6, isOver: "true/false"}
+        let theBody = {
+            act: "guess",
+            guess: `${guess}`,
+            id: `${userID}`,
```

Comment:
`guess: ${guess}` is just `guess: guess` right? 

I've never done this, but I think you can also do 
``` JavaScript
...
act: "guess",
guess,
id: userID
...
```

If we renamed `userID` as `id`, you could do it twice! THE EFFICIENCY!!!!